tests/acceptance/browzine-sersol-360-core-adapter-test.js

```diff
@@ -0,0 +1,67 @@
+describe("BrowZine SerSol 360 Core Adapter >", function() {
+  var serSol360Core = {}, searchResults = {};
+  var results = "<div ui-view='searchResults'><div class='results-title-data'><div class='results-title-row'><div class='results-title-image-div'><img src='' ng-src='' class='results-title-image'/></div><div class='results-title-details'><div class='results-identifier'>ISSN: 0028-4793</div></div></div></div></div>";
+
+  $("body").append(results);
+
+  beforeEach(function() {
+    serSol360Core = browzine.serSol360Core;
+
+    searchResults = $("div[ui-view='searchResults']");
+
+    inject(function ($compile, $rootScope) {
+      $scope = $rootScope.$new();
+
+      $scope.searchResultsCtrl = {
+        titleData: {
+          titles: [
+            {
+              title: "The New England journal of medicine",
+              syndeticsImageUrl: "https://secure.syndetics.com/index.aspx?isbn=/mc.gif&issn=0028-4793&client=mistatesum",
```

Comment:
If this is a URL that actually pings some server that maybe we shouldn't be hitting too frequently from tests, it might be good to just point it at something else, even an assets.thirdiron.com URL for some random cover, even if we label the item as `syndeticsImageUrl`

Perhaps `https://assets.thirdiron.com/default-cover-image.png?this=from-syndetics-i-guess` or something? 😊 

I just wouldn't want Syndetics to contact mistatesum and ask, "Why do we keep getting these random requests from continuous integration servers?" since I don't know what kind of Terms of Service there are for the Syndetics service.

But of course, only if this would have the possibility to truly trigger a request to that URL.  If not, then heyyy no problemo.

Reply:
hehe, yeah we're definitely not hitting that url, just parsing it. When we do the magic to get the 360 Core results data, we get back an angular scope with a property called `searchResultsCtrl`, so that whole section is just modeling that angular `$scope.searchResultsCtrl` property.

Reply:
If we don't hit it, then we're fine 👍